src/test/cli/radosgw-admin/help.t

```diff
@@ -369,11 +369,16 @@
      --rgw-obj-fs                  the field separator that will separate the rados object name from the rgw object name;
                                    additionally rados objects for incomplete multipart uploads will not be output
   
+  Bucket list objects options:
+     --max-entries                 max number of entries listed, default 1000, absolute maximum 10000
```

Comment:
So 10,000 represents how much RGW will request from the OSDs with each CLS call. If the user requests for more entries, it will just paginate 10,000 at a time until the entire list is produced. Therefore I recommend removing ", absolute maximum 10000". I apologize for not catching this earlier.

Reply:
And make the same change here. After those two changes I'll approve the PR.